packages/@azure/servicebus/data-plane/tsconfig.json

```diff
@@ -3,22 +3,22 @@
     /* Basic Options */
     "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
     "module": "es6" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-
+    "lib": [] /* lib dependencies are triple-slash directives in lib/index.ts */,
     "declaration": true /* Generates corresponding '.d.ts' file. */,
     "declarationMap": true /* Generates a sourcemap for each corresponding '.d.ts' file. */,
     "sourceMap": true /* Generates corresponding '.map' file. */,
-    
```

Comment:
The following edits are prettier deciding the default tsconfig.json is formatted improperly (the only semantic change to tsconfig is above)